en/docs/devops-and-ci-cd/autoscale/autoscale-components-with-scale-to-zero.md

```diff
@@ -17,20 +17,61 @@ Alternatively, you can enable the scale-to-zero feature for a specific environme
 
 When you turn on the scale-to-zero option for your application using the UI toggle, the minimum replicas for your app will be set to zero. However, you can still pick an appropriate maximum number of replicas. 
 
+When your application is scaled up from zero and receiving heavy traffic, Choreo will add more pods to your application's deployment. You can configure the number of requests that need to be in the pending state before Choreo add another pod to your application deployment.
```

Comment:
As your application scales from zero,  Choreo seamlessly adjusts by introducing more pods to your application's deployment. You retain the ability to fine-tune the criteria, specifying the number of pending requests that must accumulate before Choreo initiates the addition of another pod to your application deployment.